CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+1.  Update local master branch:
```

Comment:
Contributions should start from `develop` branch.

Reply:
🆗